### PR TITLE
Revert odhdashboardconfig with rhoai configuration

### PR DESCRIPTION
**Pull request by lucferbux**
Closes https://issues.redhat.com/browse/RHOAIENG-4159

## Description

Added logic to deploy custom attributes to `OdhDashboardConfig` for rhoai.

## In rhoai

1. Stop the rhoai operator
2. Remove `OdhDashboardConfig` resource from the cluster
3. Deploy this version of the odhdashboardconfig with `oc apply -k manifests/overlays/odhdashboardconfig`
5. Go to the rhoai dashbaord
6. Check that you can spin up a jupyter notebook

## Request review criteria:

Self checklist (all need to be checked):
- [X] The developer has manually tested the changes and verified that the changes work
- [X] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

**Comment by lucferbux** [comment]
@alexcreasy Part of this test would be spin up a notebook too, since it was the symptom of the issue, see https://issues.redhat.com/browse/RHOAIENG-4159

File: backend/src/utils/constants.ts [hunk 1]
```diff
@@ -58,12 +62,17 @@ export const blankDashboardCR: DashboardConfig = {
...
     notebookController: {
       enabled: true,
+      ...(isRHOAI && {
+        pvcSize: '20Gi',
+        notebookNamespace: 'rhods-notebooks',
+      }),
     },
     groupsConfig: {
-      adminGroups: 'odh-admins',
+      adminGroups: isRHOAI ? 'rhoai-admins' : 'odh-admins',
```
Comment (zdtsw) [inline backend/src/utils/constants.ts:75]: so we do not not use `dedicated-admins` ?

Comment (andrewballantyne) [reply backend/src/utils/constants.ts:75]: @zdtsw what do you guys do behind here?
https://github.com/red-hat-data-services/odh-manifests/blob/master/odh-dashboard/odhdashboardconfig/odhdashboardconfig.yaml#L27

Comment (lucferbux) [reply backend/src/utils/constants.ts:75]: If you were replacing this with dedicated-admins i can change it. Based on a conversation on slack ai thought it was changed to rhoai-admins.

Comment (VaishnaviHire) [reply backend/src/utils/constants.ts:75]: @andrewballantyne We previously defined them here https://github.com/red-hat-data-services/rhods-operator/pull/194/files#diff-3ddcc8f65a628050fbf81b0ede482ebe52b1418631bf8586c509916b8b767dcdL215

Essentially depending on the platform set the values of admins-group and then replace it in yaml file

Comment (andrewballantyne) [reply backend/src/utils/constants.ts:75]:
```
		deploy.SelfManagedRhods: "rhods-admins",
		deploy.ManagedRhods:     "dedicated-admins",
```
Ah -- there it is -- two types of deployments

Comment (lucferbux) [reply backend/src/utils/constants.ts:75]: Mmm yes, then i think it's better to keep that logic, I'm gonna revert the changes and i can sync with @zdtsw first thing in the morning to get both PRs

Comment (lucferbux) [reply backend/src/utils/constants.ts]: Ok @zdtsw I've recovered the old `odhDashboardConfig` from rhoai, it should be the same in this route:

`manifests/overlays/odhdashboardconfig`

**Review by dpanshug — APPROVED** [review]
/lgtm

**Pull request merged**